packages/common-widgets/src/email-form/widget.tsx

```diff
@@ -43,16 +44,43 @@ const Widget = ({
     dispatch,
 }: WidgetProps) => {
     const [email, setEmail] = useState("");
+    const [turnstileToken, setTurnstileToken] = useState<string>("");
+    const [errorMessage, setErrorMessage] = useState<string>("");
+
     const justifyContent =
         alignment === "center"
             ? "center"
             : alignment === "right"
             ? "flex-end"
             : "flex-start";
 
+    useEffect(() => {
+        (window as any).turnstileCallback = async (token: string) => {
```

Comment:
This should also be enabled if `state.config.turnstileKey` is defined.

Reply:
Resolved